src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -183,26 +183,26 @@ export function ConfirmationModalContent({
   bottomContent?: () => ReactNode | undefined
 }) {
   return (
-    <Wrapper>
-      <Section>
-        <RowBetween>
-          <Text fontWeight={500} fontSize={16}>
+    <ConfirmationModalWrapper>
+      <AutoColumn gap="0.5rem" padding="1em 0.5rem 0">
```

Comment:
nit: I already wrote it down somewhere, but I would consider `<AutoColumn gap="8px" padding="16px 8px 0">` much more readable and actually consistent with the rest of the codebase.

Reply:
<img width="148" alt="Screenshot 2023-04-27 at 10 19 48" src="https://user-images.githubusercontent.com/2464966/234776328-97f5415d-d48d-470c-9e8b-e92408239d80.png">
Then, you can further replace that with `grid="sm"` which is equivallent to `8px`. Before I got moved from `web` team, I wanted to do a find&replace in the entire codebase to get rid of arbitrary values, so we're more aligned with Figma.